app/controllers/dashboard/flickr_controller.rb

```diff
@@ -0,0 +1,14 @@
+require 'flickraw'
+
+class Dashboard::FlickrController < Dashboard::BaseController
+  respond_to :js
+
+  def get_flickr_images
+    FlickRaw.api_key = ENV['FLICKR_KEY']
+    FlickRaw.shared_secret = ENV['FLICKR_SECRET']
+
+    @flickr_images = flickr.photos.search(
+        extras: 'url_sq, url_m', per_page: 10, page: 1, license: '1, 2, 3, 4, 5, 6, 7',
```

Comment:
Line is too long. [87/80]<br>Indent the first parameter one step more than the previous line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.